examples/configs/evaluation/hf_evaluate_config.yaml

```diff
@@ -0,0 +1,65 @@
+name: "lm-buddy-hf-evaluate"
+
+# Input dataset path
+dataset:
+  path: "s3://platform-storage/datasets/dialogsum"
+
+
+# Settings specific to the hf_evaluate entrypoint
+evaluation:
+  # metrics to be used for the evaluation
+  # (you can add "rouge", "meteor", and "bertscore" atm)
+  metrics: ["rouge", "meteor"]
```

Comment:
should we set this to the full list of metrics as a default?

Reply:
Yeah, my bad, bertscore is slower and I left it out while heavily testing, then forgot to put it back before committing 😅 - I'll bring it back!